client/my-sites/domains/domain-management/settings/cards/strings.tsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslate } from 'i18n-calypso';
+
+() => {
+	const translate = useTranslate();
+	translate( 'Advanced settings' );
+	translate( 'Redirect type' );
+	translate( 'Select the HTTP redirect type' );
+	translate( 'Temporary redirect' );
```

Comment:
I removed the code from the string, since we'll probably use 301 and 307

Reply:
Sounds good! We could also include the translation with the 301/307 code in the string, to give us the option to include it.